crawler/features_crawler.py

```diff
@@ -800,6 +867,8 @@ def _crawl_packages(self, dbpath=None, root_dir='/'):
             distro = ''
 
             reload_needed = True
+        #TODO: Following never gets called, is this the intention? 
+        #Maybe crawl_mode needs to be set in crawl_packages()
```

Comment:
you mean MOUNTPOINT? it should be called in MOUNTPOINT mode, is that not the case?